lib/pronto/swiftlint/wrapper.rb

```diff
@@ -5,7 +5,7 @@ module Pronto
   module Swiftlint
     class Wrapper
       def lint
-        stdout, stderr, _ = Open3.capture3(swiftlint_executable)
+        stdout, stderr, _ = Open3.capture3("#{swiftlint_executable} lint --reporter json")
```

Comment:
Was gonna submit this myself.
Would love to see this merged for this change alone 😄